tweetfeed/services/dataService.js

```diff
@@ -1,44 +1,44 @@
-(function(){
+(function () {
     angular.module('myApp')
-    
-    .factory('dataService', 
-    function($http){
-        
+
+    .factory('dataService',
+    function ($http) {
+
         return {
             getTweets: getTweets
         }
-        
-        function getTweets(queryType, queryTerm){
-            
+
+        function getTweets(queryType, queryTerm) {
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*